charts/victoria-metrics-agent/templates/clusterrole.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: rbac.authorization.k8s.io/v1beta1
```

Comment:
please add configuration to values.yaml to enable/disable creation of cluster role
e.g.
```
rbac:
  create: true
```
and use this param